src/Stratis.Bitcoin.Features.Wallet/Services/WalletService.cs

```diff
@@ -1096,7 +1096,7 @@ public async Task<IActionResult> Vote(VoteRequest request, CancellationToken can
                     {
                         this.logger.LogInformation($"Casting vote {voteTransactions.IndexOf(vote)}/{voteTransactions.Count}");
 
-                        var voteCharacter = request.Vote.Value ? 1 : 0;
+                        var voteCharacter = request.Vote;
```

Comment:
Should we have a maximum length? If the vote string is too long, together with the address string, the vote transaction will be regarded as non-standard

Reply:
Good point! I think lets make it 3 characters?

Reply:
Sounds good to me 👍

Reply:
Done